CHANGELOG.md

```diff
@@ -16,6 +16,13 @@
 
 ---
 
+## v3.4.0
+
+- Migrate `serializeAttribute()` from privacy.ts to serialize.ts ([#1033](https://github.com/DataDog/browser-sdk/pull/1033))
+- Remove monitoring & ignore untrusted events ([#1029](https://github.com/DataDog/browser-sdk/pull/1029))
+- v3.3.1 ([#1031](https://github.com/DataDog/browser-sdk/pull/1031))
```

Comment:
no need to include previous release PR in the changelog

Reply:
I updated the commit

Reply:
cf release doc, you'll have to update the tag as well

Reply:
Done